src/datatable.js

```diff
@@ -211,8 +213,6 @@ var dataTable = {
         var parameters;
         var ajaxConfig;
 
-        loadingBar.start();
```

Comment:
well, I guess this should stay: `_query()` could called outside of `_refresh()`

Reply:
Yes, you're right.
Also, has @btamas pointed out, `_query` is also called from other methods...